compiler/rustc_hir/src/hir.rs

```diff
@@ -4037,8 +4038,8 @@ mod size_asserts {
     static_assert_size!(Impl<'_>, 80);
     static_assert_size!(ImplItem<'_>, 80);
     static_assert_size!(ImplItemKind<'_>, 32);
-    static_assert_size!(Item<'_>, 80);
-    static_assert_size!(ItemKind<'_>, 48);
+    static_assert_size!(Item<'_>, 88);
+    static_assert_size!(ItemKind<'_>, 56);
```

Comment:
@compiler-errors I guess this may be ungreat :). Unsure how bad would be the impact, maybe we should try to move `bidir_outlives` out from `OpaqueTy` variant.

Reply:
Yeah this is the best way of doing it.